raiden/network/discovery.py

```diff
@@ -54,13 +56,15 @@ class ContractDiscovery(Discovery):
     Allows registering and looking up by endpoint (host, port) for node_address.
     """
 
-    def __init__(self, node_address, discovery_proxy):
+    def __init__(self,
```

Comment:
the argument list should start on a new line. (there are other occurences that need fixing)